builder-support/dockerfiles/Dockerfile.rpmbuild

```diff
@@ -37,6 +37,11 @@ RUN touch /var/lib/rpm/* &&  if $(grep -q 'release 7' /etc/redhat-release); then
     fi
 @ENDIF
 
+# both options below work (right now anyway); we can get rid of this once 20211101-2 lands in epel9
+# RUN dnf install -y https://kojipkgs.fedoraproject.org//work/tasks/6683/89486683/re2-20211101-2.el9.x86_64.rpm https://kojipkgs.fedoraproject.org//work/tasks/6683/89486683/re2-devel-20211101-2.el9.x86_64.rpm
+RUN dnf install -y re2-devel && sed -i -e 's!@prefix@!/usr!' -e 's!@includedir@!/usr/include!' -e 's!@exec_prefix@!/usr!' -e 's!@libdir@!/usr/lib64!' /usr/lib64/pkgconfig/re2.pc
```

Comment:
https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2022-f063125ebb is the proposed EPEL9 update containing the fix.